### PR TITLE
Watson/add quic to openssl 3.0.12

### PR DESCRIPTION
**Pull request by wbl**

### CODE REVIEW

File: VERSION.dat [hunk 1]
```diff
@@ -2,6 +2,6 @@ MAJOR=3
 MINOR=0
 PATCH=12
 PRE_RELEASE_TAG=
-BUILD_METADATA=
-RELEASE_DATE="24 Oct 2023"
-SHLIB_VERSION=3
+BUILD_METADATA=quic
+RELEASE_DATE="24 Sep 2023"
```
Comment (tmshort) [inline VERSION.dat:6]: This should be "24 Oct 2023"

Comment (wbl) [reply VERSION.dat:6]: Nice catch. Let me know when you've reviewed the rest and I'll push a fixed version.

Comment (tmshort) [reply VERSION.dat:6]: Everything else looks good. This is all I noticed.

Comment (wbl) [reply VERSION.dat]: Fixed version pushed

**Review by kaduk — APPROVED** [review]
rebase looks clean enough to work

**Pull request merged**